src/test/java/ticTacToe/ui/UiTest.java

```diff
@@ -88,6 +88,15 @@ public void asksForCorrectStarter() {
         assertEquals("X", startingMark);
     }
 
+    @Test
```

Comment:
The name of this test has an `and` in it. This signals that it is doing two things. It might be better to have two different tests here - one that checks for a number, and one that checks it is within range. SRP applies for tests too :)